recipe/meta.yaml

```diff
@@ -41,13 +45,18 @@ requirements:
 {% set tests_to_deselect = tests_to_deselect + " --deselect=tests/test_pyproject_extra_dirs.py::test_pep517_wheel_extra_dirs" %}  # [win]
 {% set tests_to_deselect = tests_to_deselect + " --deselect=tests/test_pyproject_pep660.py::test_pep660_wheel" %}                 # [win]
 
+{% set markers_to_skip = "isolated or network or virtualenv or setuptools or broken_on_urct" %}
+
+{% set tests_to_skip = "sdist_time_hash_set_epoch or sdist_hash" %}
+
 test:
   imports:
     - scikit_build_core
   commands:
     - pip check
+    - set CMAKE_GENERATOR=Ninja  # [win]
```

Comment:
could probably set this on all platforms?

Reply:
Removing the selector for line `57` and `make` for unix on line `74`, seems to be cause similar and some new tests to start failing. I'm not 100% sure why, but looking through it.

Reply:
It seems suspicious that we have to set this here. What error did you have before setting this env var?

Reply:
@JeanChristopheMorinPerso see `win` log https://cloud.prefect.io/anaconda/flow-run/77ce1961-6c43-42bc-b97d-d20c40f2e654 of the test commit https://github.com/AnacondaRecipes/scikit-build-core-feedstock/pull/3/commits/a63fbbde4b9c988cae1b942d631fc6f7883da311

During tests (not build):
```
                   INFO           try:                                                                                                                                                                       
                   INFO               Run(env=self.env).live(self.cmake, *all_args)                                                                                                                          
                   INFO           except subprocess.CalledProcessError:                                                                                                                                      
                   INFO               msg = "CMake configuration failed"                                                                                                                                     
                   INFO   >           raise FailedLiveProcessError(msg) from None                                                                                                                            
                   INFO   E           scikit_build_core.errors.FailedLiveProcessError: CMake configuration failed                                                                                            
                   INFO                                                                                                                                                                                      
                   INFO   _cmake_args = <generator object CMaker._compute_cmake_args at 0x0000015EA7681C10>                                                                                                  
                   INFO   all_args   = ['-SC:\\b\\abs_77duo8cv0a\\croot\\scikit-build-core_1722928739052\\test_tmp\\tests\\packages\\simple_pure',                                                           
                   INFO    '-BC:\\cygwin64\\tmp\\pytest-of-builder-admin\\pytest-39\\build0\\build']                                                                                                         
                   INFO   cmake_args = ()                                                                                                                                                                    
                   INFO   defines    = None                                                                                                                                                                  
                   INFO   gen        = 'Visual Studio 16 2019 Win64'                                                                                                                                         
                   INFO   msg        = 'CMake configuration failed'                                                                                                                                          



...



                   INFO   ---------------------------- Captured stdout setup ----------------------------                                                                                                    
                   INFO   -- Building for: Visual Studio 16 2019                                                                                                                                             
                   INFO   -- The CXX compiler identification is MSVC 19.29.30154.0                                                                                                                           
                   INFO   -- Detecting CXX compiler ABI info                                                                                                                                                 
                   INFO   -- Detecting CXX compiler ABI info - done                                                                                                                                          
                   INFO   -- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/BuildTools/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x64/cl.exe - skipped                       
                   INFO   -- Detecting CXX compile features                                                                                                                                                  
                   INFO   -- Detecting CXX compile features - done                                                                                                                                           
                   INFO   -- Configuring incomplete, errors occurred!                                                                                                                                        
                   INFO   ---------------------------- Captured stderr setup ----------------------------                                                                                                    
                   INFO   CMake Error: CMAKE_GENERATOR was set but the specified generator doesn't exist. Using CMake default.                                                                               
                   INFO   ----------------------------- Captured log setup ------------------------------                                                                                                    
                   INFO   INFO     scikit_build_core:_shutil.py:69 RUN: %PREFIX%\Library\bin\cmake.EXE -E capabilities                                                                                       
                   INFO   INFO     scikit_build_core:program_search.py:84 CMake version: 3.26.4                                                                                                              
                   INFO   INFO     scikit_build_core:_shutil.py:69 RUN: %PREFIX%\Library\bin\cmake.EXE -S%SRC_DIR%\tests\packages\simple_pure                                                                
                          -BC:\cygwin64\tmp\pytest-of-builder-admin\pytest-39\build0\build                                                                                                                   
...
```